### PR TITLE
Add memory evict doc

### PR DESCRIPTION
**Pull request by zwzhang0107**

### CODE REVIEW

File: docs/user-manuals/memory-evict.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+# Eviction Strategy base on Memory Usage
+
+## Introduction
+
+Koordinator supports the dynamic overcommitment from idle resources on node to low-priority
```
Comment (saintube) [inline docs/user-manuals/memory-evict.md:5]: typo: "Koordiantor"

File: docs/user-manuals/memory-evict.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+## Use Eviction Strategy base on Memory Usage
+
+1. Create a configmap.yaml file based on the following ConfigMap content:
+   ```yaml
+   #ConfigMap slo-controller-config example。
+   apiVersion: v1
+   kind: ConfigMap
+   metadata:
+     name: slo-controller-config # name should be set as the configuration of koord-manager, e.g. ack-slo-config
```
Comment (saintube) [inline docs/user-manuals/memory-evict.md:41]: perhaps remove the naming like `ack-xxx`?

File: docs/user-manuals/memory-evict.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    
+3. Create a file named be-pod-demo.yaml based on the following YAML content:
+
+   ```yaml
+   apiVersion: v1
+   kind: Pod
+   metadata:
+     name: be-pod-demo
+     labels:
+       koordinator.sh/qosClass: 'BE' #指定Pod的QoS级别为BE。
```
Comment (saintube) [inline docs/user-manuals/memory-evict.md:81]: need to comment in English

File: docs/user-manuals/memory-evict.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+   ```
+   
+5. Run the following command to check the ls-pod-demo pod in Running state:
+
+   ```bash
+   $ kubectl get pod be-pod-demo
+   NAME          READY   STATUS    RESTARTS   AGE
+   be-pod-demo   1/1     Running   0          7s
+   ```
+6. Run the following command through [stress tool](https://linux.die.net/man/1/stress)
```
Comment (saintube) [inline docs/user-manuals/memory-evict.md:111]: typo: "mease", "adjuested"

File: docs/user-manuals/memory-evict.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+make sure the memory usage of node is above the threshold config, and the argument `--vm-bytes`
+mease the process will consume 10GB memory, this should be adjuested according to the node capacity.
+
+   ```bash
+   $ stress --cpu 1 --vm 1 --vm-bytes 10G --vm-keep
+   ```
+
+7. Check the running state of be-pod-demo, then you can find the e-pod-demo pod is not exsit,
```
Comment (saintube) [inline docs/user-manuals/memory-evict.md:119]: `e-pod-demo` -> `be-pod-demo`

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/memory-evict.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
+# 基于内存用量的驱逐策略
+
+## 简介
+
+Koordinator支持了将节点空闲资源以Batch优先级的形式动态超卖给低优先级Pod，在混部场景下，节点实际的内存资源用量时刻在变化，对于内存这类不可压缩类型的资源，
```
Comment (saintube) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/memory-evict.md:5]: 这里是否需要说明驱逐是针对 Batch 还是 BE（缺省为 Batch）pods

File: docs/user-manuals/memory-evict.md (hunk 2 above)
Comment (zwzhang0107) [reply docs/user-manuals/memory-evict.md:41]: for ack user better understanding

**Review by eahydra — APPROVED** [review]
/lgtm
/approve

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

**Pull request merged**